src/main/java/org/broadinstitute/hellbender/bench/TestMath.java

```diff
@@ -1,64 +0,0 @@
-package org.broadinstitute.hellbender.bench;
-
-/**
- * Demonstrates that adding small doubles in different orders doesn't always give the same answer.
- */
-public class TestMath {
```

Comment:
Huh, I thought jp had just added this to our wiki instead of committing it.

https://github.com/broadinstitute/gatk/wiki/Numerical-errors